apis/keda/v1alpha1/scaledobject_webhook.go

```diff
@@ -149,10 +149,15 @@ func verifyHpas(incomingSo *ScaledObject, action string) error {
 			}
 
 			if !owned {
-				err = fmt.Errorf("the workload '%s' of type '%s' is already managed by the hpa '%s'", incomingSo.Spec.ScaleTargetRef.Name, incomingSoGckr.GVKString(), hpa.Name)
-				scaledobjectlog.Error(err, "validation error")
-				prommetrics.RecordScaledObjectValidatingErrors(incomingSo.Namespace, action, "other-hpa")
-				return err
+				if incomingSo.ObjectMeta.Annotations["keda.sh/transfer-hpa-ownership"] == "true" &&
+					incomingSo.Spec.Advanced.HorizontalPodAutoscalerConfig.Name == hpa.Name {
+					scaledobjectlog.Info(fmt.Sprintf("%s hpa ownership being transferred to %s", hpa.Name, incomingSo.Name))
```

Comment:
Do we want to print a message in any case, or only in debug?
I believe that this will be executed only in a few cases so I don't think is a blocker, just a thought

Reply:
I don't have a strong opinion, I normally don't run the operator in debug mode, though, so since this is a one-time operation, I felt like `info` was appropriate. Happy to make it debug, but after thinking a bit more, I wish that was an event in the ScaledObject resource.

Reply:
Agree!